crm_rma_advance_location/stock.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright 2013 Camptocamp
+#    Copyright 2009-2013 Akretion,
+#    Author: Emmanuel Samyn, Raphaël Valyi, Sébastien Beau,
+#            Joel Grand-Guillaume
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api, SUPERUSER_ID
+from openerp.tools.translate import _
+
+
+class stock_warehouse(models.Model):
+
+    _inherit = "stock.warehouse"
+
+    lot_carrier_loss_id = fields.Many2one(
+        'stock.location',
+        'Carrier Loss Location')
+    lot_breakage_loss_id = fields.Many2one(
+        'stock.location',
+        'Breakage Loss Location')
+    lot_refurbish_id = fields.Many2one(
+        'stock.location',
+        'Refurbish Location')
+
+    def init(self, cr):
```

Comment:
Replace this with a post-init hook. See an example here:

1. https://github.com/odoomrp/odoomrp-wip/blob/8.0/quality_control_stock/__openerp__.py#L44
2. https://github.com/odoomrp/odoomrp-wip/blob/8.0/quality_control_stock/__init__.py#L10